byway/src/main/webapp/script.js

```diff
@@ -12,19 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- *
- */
-/* exported addRandomGreeting */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function initMap() {
```

Comment:
Let avoid putting all of the web apps javascript in a single file. Trying moving this logic to a new file the corresponds to your html page, or you can split it up into files based on the logic itself, or both. Whatever you think makes more sense in the context of the larger web app.

Reply:
I'll move map out to another js file next week once Rena and I have gotten a chance to talk about what we want to include in the class. But I'll change the name of script.js to be page-specific.

Reply:
Acknowledged